File: SDKs/amazonaws.com/waf-regional/2016-11-28/java/lib/src/main/java/openapisdk/models/shared/DeleteXssMatchSetRequest.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * DeleteXssMatchSetRequest
 * A request to delete an <a>XssMatchSet</a> from AWS WAF.
**/
public class DeleteXssMatchSetRequest {
    @JsonProperty("ChangeToken")
    public String changeToken;
    public DeleteXssMatchSetRequest withChangeToken(String changeToken) {
        this.changeToken = changeToken;
        return this;
    }
    @JsonProperty("XssMatchSetId")
    public String xssMatchSetId;
    public DeleteXssMatchSetRequest withXssMatchSetId(String xssMatchSetId) {
        this.xssMatchSetId = xssMatchSetId;
        return this;
    }
}

File: SDKs/amazonaws.com/waf-regional/2016-11-28/python/src/sdk/models/shared/deletexssmatchsetrequest.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class DeleteXSSMatchSetRequest:
    r"""DeleteXSSMatchSetRequest
    A request to delete an <a>XssMatchSet</a> from AWS WAF.
    """
    
    change_token: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('ChangeToken') }})
    xss_match_set_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('XssMatchSetId') }})
    
